package.json

```diff
@@ -27,21 +28,21 @@
   "homepage": "https://github.com/senecajs-labs/seneca-sqlite-store",
   "bugs": "https://github.com/senecajs-labs/seneca-sqlite-store/issues",
   "dependencies": {
-    "sqlite3":    ">=2.1.5",
-    "node-uuid":  "*",
     "eraro": "^0.4.1",
-    "lodash": "^4.3.0"
+    "lodash": "^4.3.0",
+    "node-uuid": "*",
+    "sqlite3": ">=2.1.5"
```

Comment:
Q: Would it be okay to freeze these dependencies?

Reply:
yes please